engine/src/known_types.rs

```diff
@@ -191,6 +191,11 @@ impl TypeDatabase {
         self.canonical_names.keys().chain(self.by_rs_name.keys())
     }
 
+    /// Returns whether the given type is a known type.
+    pub(crate) fn is_known(&self, qn: &QualifiedName) -> bool {
```

Comment:
Is this not a duplicate of `is_known_type`?

Reply:
Yep, removed.